### PR TITLE
QoL Changes

### PR DESCRIPTION
**Pull request by GenelleGeerman**
# Context
Added some Quality of Life changes to better test things later on.

# Additions
* A UI For Game Over (added through player spawner)
* A playtest scene
* Preset PlayerShip

# Modified
* Start Menu has 2 new buttons: Test and Delete Save
    * Test is for the playtests
    * Delete Save removes the old ship save in case of errors
* Game.cs now has a IsUiOpen bool to disable and enable player controls
* PlayerController checks is UI is open to be able to move.
* Old ShipBuilder has a color picker now
* Station.cs -> Added enabling and disabling of UI from Game
# Bug Fix
Fixed an issue with tilesets due to reimports

### CODE REVIEW

File: Assets/Structures/Tilesets/TilesNoCollision.tres
Comment (BenDBrown) [inline Assets/Structures/Tilesets/TilesNoCollision.tres:1]: I see in bugfix why this is changed by preferably mention something like changes tile collision resources under the modified heading

Comment (GenelleGeerman) [reply Assets/Structures/Tilesets/TilesNoCollision.tres:1]: The bugfix was a reimport so I placed it there. I didn't touch it other than reopening it

File: Structures/Station.cs
Comment (BenDBrown) [inline Structures/Station.cs:1]: please mention this in modified, I understand its part of the stopping movement when UI open stuff, but having a list of modified feature groups in this case station would make it easier to track down why if station suddenly broke cuz of this or smthn.

**Review by BenDBrown — APPROVED** [review]
looks good, more detail in modified next time please

File: Structures/Station.cs
Comment (GenelleGeerman) [reply Structures/Station.cs:1]: Comment Added

**Pull request merged**